wundertools/docker/alpine-mariadb/Makefile

```diff
@@ -0,0 +1,31 @@
+IMAGE = alpine-mariadb
+
+build: conf.database conf.user conf.root.pass conf.user.pass
+	docker \
+		build \
+		--build-arg "MYSQL_ROOT_PASSWORD=$(shell cat conf.root.pass)" \
+		--build-arg "MYSQL_DATABASE=$(shell cat conf.database)" \
+		--build-arg "MYSQL_USER=$(shell cat conf.user)" \
+		--build-arg "MYSQL_PASSWORD=$(shell cat conf.user.pass)" \
+		-t "$(IMAGE)" \
+		.
+
+rmi:
+	-docker rmi $(IMAGE)
+
+clean:
+	docker ps -a --filter status="exited" --format '{{.ID}}' | xargs -r docker rm
+	docker images --filter "dangling=true" -q | xargs -r docker rmi
+	rm -f conf.*
+
```

Comment:
All targets do something. These are things which I use regularly so they have practical background. If you do a lot of image building you see that you would use these things often.